### PR TITLE
Hue: Update README.md

### PR DESCRIPTION
**Pull request by wayneoutthere**
Attempted to update 2 URLS to dead links in Motion sensor and dimmer switch

### CODE REVIEW

File: zigbeephilipshue/README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ The [Hue outdoor sensor](https://www.philips-hue.com/en-us/p/hue-outdoor-sensor/
 
 ### Hue dimmer switch
 
-The [Hue dimmer switch](https://www.philips-hue.com/en-us/p/hue-dimmer-switch--latest-model-/046677562779) (including the older model version) is fully supported.
+The [Hue dimmer switch](https://www.philips-hue.com/en-us/p/hue-dimmer-switch--latest-model-/046677562779  https://www.philips-hue.com/en-us/p/hue-dimmer-switch--latest-model-/046677562779) (including the older model version) is fully supported.
```
Comment (mzanetti) [inline zigbeephilipshue/README.md:18]: I think a MD link tag can't have 2 links. Besides, isn't it the same link twice? Looks like a copy/paste mistake maybe?

**Review by mzanetti — CHANGES_REQUESTED** [review]
There's one issue with the dimmer switch link. the others look fine. Thanks!

File: zigbeephilipshue/README.md (hunk 1 above)
Comment (wayneoutthere) [inline zigbeephilipshue/README.md]: hows' that?

**Review by mzanetti — APPROVED** [review]

**Review by mzanetti — APPROVED** [review]

**Comment by wayneoutthere** [comment]
nice! you've built up my self worth, self esteem and will-to-live! ;)

**Pull request merged**